sklearnex/basic_statistics/__init__.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+from .basic_statistics import BasicStatistics
+
+__all__ = ['BasicStatistics']
```

Comment:
Do we leaving this kind of functionality in `sklearnex.basic_statistics`? Are we planning to move this kind of new functionality, that is not in sklearn API to the new namespace in `sklearnex`?

Reply:
I believe we have had an agreement on this topic. The thing is that we do not have any obligations on such functionality since it does not have any counterparts in the vanilla scikit-learn. Thus we can introduce pretty much anything in this namespace as long as it only extends existing API.